Marlin/Configuration.h

```diff
@@ -156,8 +156,20 @@
 //#define SWITCHING_EXTRUDER
 #if ENABLED(SWITCHING_EXTRUDER)
   #define SWITCHING_EXTRUDER_SERVO_NR 0
-  #define SWITCHING_EXTRUDER_SERVO_ANGLES { 0, 90 } // Angles for E0, E1
-  //#define HOTEND_OFFSET_Z {0.0, 0.0}
+  #define SWITCHING_EXTRUDER_SERVO_ANGLES { 17, 167 } // Angles for E0, E1
+  #define HOTEND_OFFSET_Z {0.0, 0.0}
+#endif
+
+// A single extruder that uses one or two single stepper motor
+// Don't forget to set SSDE_SERVO_ANGLES 
+#define SINGLENOZZLE_SWITCHING_EXTRUDER
```

Comment:
This file is the base/demo configuration for all users. So, you should include only these changes, adding the new extruder type. And `SINGLENOZZLE_SWITCHING_EXTRUDER` must be disabled by default.